stdlib/filename.mli

```diff
@@ -183,9 +183,21 @@ val get_temp_dir_name : unit -> string
 (** The name of the temporary directory:
     Under Unix, the value of the [TMPDIR] environment variable, or "/tmp"
     if the variable is not set.
-    Under Windows, the value of the [TEMP] environment variable, or "."
-    if the variable is not set.
+
+    Under Windows, it uses [GetTempPath2] (since Windows 10 21H1), or
+    [GetTempPath], and returns:
+    - if the process runs as SYSTEM, the path "C:\\Windows\\SystemTemp\\"
+      (since Windows 10 21H1); or
```

Comment:
It is not very clear what `(since ...)` refers to. I suggest something like:
```suggestion
    - if the process runs as SYSTEM in Windows 10 21H1 or later, the path "C:\\Windows\\SystemTemp\\"; or
```

Reply:
LGTM.